app/controllers/admins/admins_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Admins
+  class AdminsController < ApplicationController
+    before_action :set_admin
+
+    def edit_password; end
+
+    def update_password
+      if @admin.update_with_password(pass_params)
+        redirect_to('/admins/users')
```

Comment:
`'/admins/users'` no-no-no
rails provide you with dynamic url helper, pls, use Rails way to determine path

Reply:
solved this by adding `admins_users_path`